client/components/basic/userStatus/UserStatus.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Box } from '@rocket.chat/fuselage';
+
+const statusColors = {
+	offline: 'neutral-500',
+	busy: 'danger-500',
+	away: 'warning-500',
+	online: 'success-500',
+};
```

Comment:
This enum could be a `export default` in a standalone module. I think other components will benefit from it.